pkgs/applications/science/misc/openmvg/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib, stdenv, fetchgit, pkgconfig, cmake, gnumake, glibc
+, enableJpeg ? false, libjpeg
```

Comment:
Actually, you can even check `(libjpeg == null)` to determine the flags.

OpenCV has a few flags not literally mapped to dependency libraries (`enableContrib`, `enableIpp`)